fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/security/PublicKey.java

```diff
@@ -26,6 +26,11 @@ public PublicKey(String data) {
     verifier = new Ed25519Verify(this.data);
   }
 
+  public PublicKey(PublicKey organizer) {
+    super(organizer.data); // Deep copy of byte array is done in parent constructor
+    verifier = new Ed25519Verify(organizer.data);
```

Comment:
```suggestion
    verifier = new Ed25519Verify(this.data);
```
should we use this, then, if the deep-copy happened in the parent constructor ?